src/main/java/by/exadel/internship/dto/formDTO/UserRoleDTO.java

```diff
@@ -0,0 +1,14 @@
+package by.exadel.internship.dto.formDTO;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class UserRoleDTO {
```

Comment:
Is this  DTO used anywhere? If no, remove it please.

Reply:
done